_docs/_latest/configure/cloud-profiler/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Cloud Profiler
+order: 600
+---
+
+# {{ page.title }}
+
+[Cloud Profiler](https://cloud.google.com/profiler/) is a statistical, low-overhead profiler on GCP that continuously 
+gathers CPU usage and memory-allocation information from your Forseti application, helping you identify the parts of 
+the application consuming the most resources and providing a more complete picture of the application's performance.
+
+---
+
+## Setting up
+
+In your `main.tf` file, set the `cloud_profiler_enabled` variable in the Forseti Terraform module to `true`:
+
+```
+module "forseti" {
+  source                   = "terraform-google-modules/forseti/google"
+  project_id               = "PROJECT_ID"
+  org_id                   = "ORG_ID"
+  domain                   = "DOMAIN"
+  
+  ...
+  
+  cloud_profiler_enabled   = true
+}
+```
+
+Apply the Terraform module.
+
+```
+terraform apply
+```
+
+## Permissions & Installations
```

Comment:
I think it would be awesome to be clear that when people are done with troubleshooting, they should disable it.  Otherwise, there will be processing overhead that will slow down Forseti.